dist/panel-triggers/module.js

```diff
@@ -9784,7 +9784,13 @@ var AlertCard = /** @class */ (function (_super) {
         };
         _this.onLinkIconClick = function (event, url) {
             event.stopPropagation();
-            window.top.location.href = url;
+            // Add 'foldTabRow' query param so that Grafana tab row panel keeps folded
+            var urlBase = url.substring(0, url.indexOf('?') > -1 ? url.indexOf('?') : url.length);
+            var urlQuery = url.substring(url.indexOf('?') > -1 ? url.indexOf('?') : url.length);
+            var queryObj = _this.parseParamsStringToObject(urlQuery);
+            queryObj['foldTabRow'] = 'true';
+            var newQuery = _this.parseParamsObjectToString(queryObj);
```

Comment:
Technically converting object to string would be formatting not parsing. 😄 

This function could also be implemented as member of URL object mentioned above (e.g. url.getQueryString()).

Reply:
Yep